src/site/xdoc/checks/coding/matchxpath.xml

```diff
@@ -48,7 +48,7 @@
               <td><a id="query"/><a href="#query">query</a></td>
               <td>Specify Xpath query.</td>
               <td><a href="../../property_types.html#String">String</a></td>
-              <td><code>&quot;&quot;</code></td>
+              <td><code></code></td>
```

Comment:
this actually good

<img width="1377" height="533" alt="Image" src="https://github.com/user-attachments/assets/5f83741f-6353-494c-a4a3-fe8daf9295b9" />